docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -1315,6 +1315,40 @@ When specifying file names, Teku expects that the files exist.
 
 List of validator public keys used by an external signer (for example, Web3Signer).
 
+### validators-external-signer-slashing-protection-enabled
+
+=== "Syntax"
+
+    ```bash
+    --validators-external-signer-slashing-protection-enabled[=<BOOLEAN>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --validators-external-signer-slashing-protection-enabled=false
+    ```
+
+=== "Environment Variable"
+
+    ```bash
+    TEKU_VALIDATORS_EXTERNAL_SIGNER_SLASHING_PROTECTION_ENABLED=false
+    ```
+
+=== "Configuration File"
+
+    ```bash
+    validators-external-signer-slashing-protection-enabled: false
+    ```
+
+Specify whether to use Teku's built-in [slashing protection] when using an external signer such as
+[Web3Signer]. Defaults to `true`.
+
+Set this option to `false` if using the slashing protection implemented by an external signer.
+
+If using Teku to sign blocks and attestations, then Teku will always use its built-in slashing
+protection.
```

Comment:
You can have a mix of internal and external signers so maybe something like:

```suggestion
Built-in slashing protection can only be disabled for validators using external signers. Validators using Teku to sign blocks and attestations will always use its built-in slashing protection.
```

Reply:
Done.